kahuna/public/js/components/gu-lazy-table/gu-lazy-table-cell.js

```diff
@@ -17,8 +17,8 @@ lazyTableCell.directive('guLazyTableCell',
         transclude: true,
         template: '<ng-transclude ng:if="guLazyTableCellVisible"></ng-transclude>',
         link: function(scope, element, attrs, ctrl) {
-            const item = scope.$eval(attrs.guLazyTableCell);
-            const position$ = ctrl.getItemPosition$(item);
+            const item$ = observe$(scope, attrs.guLazyTableCell);
+            const position$ = item$.flatMapLatest((item) => ctrl.getItemPosition$(item));
```

Comment:
Note I think this could be simplified to `item$.flatMapLatest(ctrl.getItemPosition$)`